ansible/monitoring.yml

```diff
@@ -118,3 +126,20 @@
 
   roles:
     - { role: Datadog.datadog }
+
+  post_tasks:
+    - name: Copy datadog.conf to rsyslog
+      copy:
+        src: files/rsyslog/datadog.conf
+        dest: /etc/rsyslog.d/datadog.conf
```

Comment:
Who guarantees that rsyslog is actually installed?

I see OS images are:

https://github.com/aeternity/infrastructure/blob/b1bb61176a5457970162fba36656240206aeea30/ansible/environments.yml#L60

https://github.com/aeternity/infrastructure/blob/b1bb61176a5457970162fba36656240206aeea30/ansible/environments.yml#L137

that seem to be "ubuntu 16.04.3 with master user + python added".

Reply:
Ubuntu versions that we use have rsyslog pre-installed.

Reply:
While `infrastructure/ansible/environments.yml` is still in use, it's deprecated and I don't plan to support it. We're migrating to AWS/GCP soon, and it well be deleted in https://www.pivotaltracker.com/story/show/160761831